config/core/build-configs.yaml

```diff
@@ -733,9 +736,12 @@ build_configs:
       gcc-8:
         build_environment: gcc-8
         architectures:
-          arm: *arm_arch
-          arm64: *arm64_arch
-          x86_64: *x86_64_arch
+          arm:
+            base_defconfig: 'cip://4.19.y-cip-rt/arm/qemu_arm_defconfig'
+          arm64:
+            base_defconfig: 'cip://4.19.y-cip-rt/arm64/qemu_arm64_defconfig'
+          x86_64:
+            base_defconfig: 'cip://4.19.y-cip-rt/x86/cip_qemu_defconfig'
```

Comment:
These -rt configs don't seem to exist:
```
$ curl -sI https://gitlab.com/cip-project/cip-kernel/cip-kernel-config/-/raw/master/4.19.y-cip/x86/cip_qemu_defconfig | head -1
HTTP/2 200
$ curl -sI https://gitlab.com/cip-project/cip-kernel/cip-kernel-config/-/raw/master/4.19.y-cip-rt/x86/cip_qemu_defconfig | head -1
HTTP/2 404
```
There appears to be only some Siemens configs in the repository:
https://gitlab.com/cip-project/cip-kernel/cip-kernel-config/-/tree/master/4.19.y-cip-rt/x86

So maybe drop the -rt ones for now in this PR and make another one later when the config files are there?

Reply:
See this fixup PR: https://github.com/aliceinwire/kernelci-core/pull/2

Please feel free to merge it on top of this branch if you're OK with the changes.